### ISSUE TITLE
When will AllenNLP v1.0 be live ?

### ISSUE THREAD

**Issue by datix-vishu** [allenai/allennlp#4119]
@dirkgr Is there a fine-grained-NER model for AllenNLP v1.0rc1 ?

**Comment by datix-vishu** [comment]
@dirkgr I think you guys made a small mistake in the demo site "Usage" tab for NER. This is how it is given in the demo site.

```
from allennlp.predictors.predictor import Predictor
import ner.crf_tagger
```


It should be:


```
from allennlp.predictors.predictor import Predictor
import allennlp_models.ner.crf_tagger
```

**Comment by dirkgr** [comment]
@datix-vishu, I cannot promise that I'll have time to fix the fine grained NER model before 1.0 goes live. There are a ton of issues open right now. I would be very grateful if you gave it a go yourself. If you can make it work, send me a PR and we'll get it in?

**Comment by dirkgr** [comment]
You are right about the second issue. I put a PR at https://github.com/allenai/allennlp-demo/pull/369.
